src/main/asciidoc/_chapters/schema_design.adoc

```diff
@@ -1142,7 +1142,10 @@ Disable Nagle’s algorithm. Delayed ACKs can add up to ~200ms to RPC round trip
 Detect regionserver failure as fast as reasonable. Set the following parameters:
 
 * In `hbase-site.xml`, set `zookeeper.session.timeout` to 30 seconds or less to bound failure detection (20-30 seconds is a good start).
-- Notice: the `sessionTimeout` of zookeeper is limited between 2 times and 20 times the `tickTime`(the basic time unit in milliseconds used by ZooKeeper.the default value is 2000ms.It is used to do heartbeats and the minimum session timeout will be twice the tickTime).
+- Note: Zookeeper clients negotiate a session timeout with the server during client init. Server enforces this timeout to be in the
+range of 2 times `tickTime` and 20 times `tickTime` unless `minSessionTimeout` and `maxSessionTimeout` are configured in the Zookeeper
```

Comment:
You should also explain what `minSessionTimeout` and `maxSessionTimeout` offer in alternative.

Reply:
Reworded it slightly, LMKWYT.